src/main/java/ddingdong/ddingdongBE/domain/notice/service/FacadeAdminNoticeService.java

```diff
@@ -0,0 +1,94 @@
+package ddingdong.ddingdongBE.domain.notice.service;
+
+import static ddingdong.ddingdongBE.domain.fileinformation.entity.FileDomainCategory.NOTICE;
+import static ddingdong.ddingdongBE.domain.fileinformation.entity.FileTypeCategory.FILE;
+import static ddingdong.ddingdongBE.domain.fileinformation.entity.FileTypeCategory.IMAGE;
+
+import ddingdong.ddingdongBE.domain.fileinformation.entity.FileInformation;
+import ddingdong.ddingdongBE.domain.fileinformation.service.FileInformationService;
+import ddingdong.ddingdongBE.domain.notice.entity.Notice;
+import ddingdong.ddingdongBE.domain.notice.service.dto.command.CreateNoticeCommand;
+import ddingdong.ddingdongBE.domain.notice.service.dto.command.UpdateNoticeCommand;
+import ddingdong.ddingdongBE.file.FileStore;
+import ddingdong.ddingdongBE.file.service.FileService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.multipart.MultipartFile;
+
+@RequiredArgsConstructor
+@Service
+@Transactional(readOnly = true)
+public class FacadeAdminNoticeService {
+
+    private final NoticeService noticeService;
+    private final FileService fileService;
+    private final FileInformationService fileInformationService;
+    private final FileStore fileStore;
+
+    @Transactional
+    public void create(CreateNoticeCommand command) {
+        Notice notice = command.toEntity();
+        Notice savedNotice = noticeService.save(notice);
```

Comment:
p4) saved notice id만 필요하다면 이 필드는 제거하는 건 어떨까요?

```suggestion
        Notice savedNotice = noticeService.save(notice);
```

Reply:
이 필드는 필요할 것 같습니다! toEntity로 생성된 notice는 아이디가 없고, 저장되고 난 후의 엔터티의 아이디가 필요하기 때문입니다!

save() 리턴 타입으로 id를 반환하지 않은 이유는 저장 후 해당 엔터티의 id를 반환하는 것보다 엔터티를 반환하는 것이 덜 어색하다고 생각했습니다